proto/all_test.go

```diff
@@ -1860,6 +1860,21 @@ func TestRequiredNotSetError(t *testing.T) {
 	}
 }
 
+func TestRequiredNotSetErrorWithBadWireTypes(t *testing.T) {
+	// Required field is properly a varint.
+	if err := Unmarshal([]byte{0x08, 0x00}, new(GoEnum)); err != nil {
+		t.Errorf("Unmarshal = %v, want nil", err)
+	}
+	// Required field is a fixed32 instead of varint.
```

Comment:
I got confused a bit on the comment.  How about...

// Bytes contain fixed32 for the required varint field.